build/steps-build-android.yml

```diff
@@ -14,6 +14,12 @@ parameters:
   type: string
 - name: googleServicesJsonFile
   type: string
+- name: artifactName
```

Comment:
Is this required because I don't see any other changes?

Reply:
yes, it's used on line 115 on the "Publish Artifact" task